tests/KubernetesClient.Tests/KubernetesClient.Tests.csproj

```diff
@@ -1,23 +1,35 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <RootNamespace>k8s.tests</RootNamespace>
+    <RootNamespace>k8s.Tests</RootNamespace>
     <TargetFrameworks>netcoreapp2.1;netcoreapp2.0</TargetFrameworks>
+    <LangVersion>8</LangVersion>
+      <SignAssembly>true</SignAssembly>
+      <AssemblyOriginatorKeyFile>..\..\src\KubernetesClient\kubernetes-client.snk</AssemblyOriginatorKeyFile>
```

Comment:
I'm unit testing internal methods / classes, so needed to make test project friend of main assembly. But can't do that without signing test project since main is signed